hack/tools/go.mod

```diff
@@ -3,9 +3,10 @@ module github.com/openshift/cluster-monitoring-operator/hack
 go 1.14
 
 require (
-	github.com/brancz/gojsontoyaml v0.0.0-20191212081931-bf2969bbd742
+	github.com/brancz/gojsontoyaml v0.0.0-20200602132005-3697ded27e8c
 	github.com/campoy/embedmd v1.0.0
 	github.com/google/go-jsonnet v0.16.1-0.20200703153429-aaf50f5b655f
 	github.com/jsonnet-bundler/jsonnet-bundler v0.4.0
-	github.com/prometheus/prometheus v1.8.2-0.20200329163539-7646cbca3282
+	github.com/prometheus/prometheus v1.8.2-0.20201015110737-0a7fdd3b7696
+	gopkg.in/yaml.v2 v2.4.0 // indirect
```

Comment:
What's weird is that gojsontoyaml and latest prometheus use yaml.v2 v2.3.0 so there shouldn't be any collision

Reply:
Even when I tried to bump to the latest version of gojsontoyaml, its yaml.v2 dep was overriden (I guess) by the prometheus one.
A solution to avoid this collision was to have the following replace statement, although I don't think its a valid solution to our problem.
```go
replace gopkg.in/yaml.v2 => gopkg.in/yaml.v2 v2.1.1
```
Do you have any suggestion to prevent this collision?

Reply:
Resolving this discussion as @simonpasquier suggested a better approach here: https://github.com/openshift/cluster-monitoring-operator/pull/1021#discussion_r543356127